docs/en/04-supported-codebases/mmcls.md

```diff
@@ -1,20 +1,163 @@
+
+```
+classification_{backend}-{precision}_{static | dynamic}_{shape}.py
+```
+
+- **{backend}:** inference backend, such as onnxruntime, tensorrt, pplnn, ncnn, openvino, coreml and etc.
+- **{precision}:** fp16, int8. When it's empty, it means fp32
+- **{static | dynamic}:** static shape or dynamic shape
+- **{shape}:** input shape or shape range of a model
+
+Therefore, in the above example, you can also convert `resnet18` to other backend models by changing the deployment config file `classification_onnxruntime_dynamic.py` to [others](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmcls), e.g., converting to tensorrt-fp16 model by `classification_tensorrt-fp16_dynamic-224x224-224x224.py`.
+
+```{tip}
+When converting mmcls models to tensorrt models, --device should be set to "cuda"
+```
+
+## Model Specification
+
+Before moving on to model inference chapter, let's know more about the converted model structure which is very important for model inference.
+
+The converted model locates in the working directory like `mmdeploy_models/mmcls/ort` in the previous example. It includes:
+
+```
+mmdeploy_models/mmcls/ort
+├── deploy.json
+├── detail.json
+├── end2end.onnx
+└── pipeline.json
+```
+
+in which,
+
+- **end2end.onnx**: backend model which can be inferred by ONNX Runtime
+- **deploy.json**: meta information about backend model
+- **pipeline.json**: inference pipeline of mmdeploy SDK
+- **detail.json**: conversion parameters
+
+The whole package **mmdeploy_models/mmcls/ort** is defined as **mmdeploy SDK model**, i.e., **mmdeploy SDK model** includes both backend model and inference meta information.
+
+## Model inference
+
+### Backend model inference
+
+MMDeploy provides a unified API named as `inference_model` to inference model, making all inference backends API transparent to users.
+
+Take the previous converted `end2end.onnx` model as an example, you can use the following code to inference the model.
+
+```shell
+from mmdeploy.apis import inference_model
+result = inference_model(
+  model_cfg='./resnet18_8xb32_in1k.py',
+  deploy_cfg='configs/mmcls/classification_onnxruntime_dynamic.py',
+  backend_files=['mmdeploy_models/mmcls/ort/end2end.onnx'],
+  img='tests/data/tiger.jpeg',
+  device='cpu')
+print(result)
+```
+
+### SDK model inference
+
+You can also perform SDK model inference like following,
+
+```python
+from mmdeploy_python import Classifier
+import cv2
+
+img = cv2.imread('tests/data/tiger.jpeg')
+
+# create a classifier
+classifier = Classifier(model_path='./mmdeploy_models/mmcls/ort', device_name='cpu', device_id=0)
+# perform inference
+result = classifier(img)
+# show inference result
+for label_id, score in result:
+    print(label_id, score)
+```
+
+Besides python API, mmdeploy SDK also provides other FFI (Foreign Function Interface), such as C, C++, C#, Java and so on. You can learn their usage from [demos](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/demo).
+
+## Supported models
```

Comment:
Should we keep this table? We already have such table in benchmark.

Reply:
> Suggest using picture links rather than PNG images to reduce the size of the repo.

THose images are used as input images when converting pytorch models

Reply:
Due to the hard maintenance of benchmark.md, we are planning to maintain the benchmark in each codebase's deployment document